{{cookiecutter.project_slug}}/requirements/dev.txt

```diff
@@ -1,10 +1,14 @@
 -r test.txt
```

Comment:
Please get rid of these changes, we can put them in the other PR :beach_umbrella: